collab-database/src/rows/row.rs

```diff
@@ -547,24 +552,24 @@ impl Default for CreateRowParams {
   fn default() -> Self {
     Self {
       id: gen_row_id(),
-      cells: Default::default(),
+      cells: Cells::default(),
       height: 60,
       visibility: true,
       row_position: OrderObjectPosition::default(),
-      timestamp: 0,
+      created_at: 0,
+      modified_at: 0,
     }
   }
 }
 
 impl CreateRowParams {
   pub fn new(id: RowId) -> Self {
+    let timestamp = timestamp();
     Self {
       id,
-      cells: Cells::default(),
-      height: 60,
-      visibility: true,
-      row_position: OrderObjectPosition::default(),
-      timestamp: timestamp(),
+      created_at: timestamp,
+      modified_at: timestamp,
+      ..Default::default()
```

Comment:
using default here will cause the heigh be zero

Reply:
Both visibility and height are unused fields, that's why I didn't pay particular attention to them. On the frontend, the rows take up as much space as they need (or if we introduce a don't wrap feature, that will also be set in field settings). To be more explicit, I can revert this change.

Reply:
So the frontend has default height for row. it's ok or we can remove the height